.cypress/utils/app_constants.js

```diff
@@ -3,31 +3,27 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { suppressResizeObserverIssue } from './constants';
-
 export const delay = 1000;
 export const timeoutDelay = 30000;
-export const TYPING_DELAY = 500;
+export const TYPING_DELAY = 150;
 
 export const moveToHomePage = () => {
-  cy.visit(`${Cypress.env('opensearchDashboards')}/app/observability-dashboards#/application_analytics/`);
+  cy.visit(`${Cypress.env('opensearchDashboards')}/app/observability-applications#/`);
   cy.wait(delay * 3);
   cy.get('.euiTitle').contains('Applications').should('exist');
 };
 
 export const moveToCreatePage = () => {
-  cy.visit(`${Cypress.env('opensearchDashboards')}/app/observability-dashboards#/application_analytics/`);
-  cy.get('.euiButton[href="#/application_analytics/create"]').eq(0).click();
-  suppressResizeObserverIssue();
+  cy.visit(`${Cypress.env('opensearchDashboards')}/app/observability-applications#/`);
+  cy.get('.euiButton[href="#/create"]').eq(0).click();
   cy.get('[data-test-subj="createPageTitle"]').should('contain', 'Create application');
 };
 
 export const moveToApplication = (name) => {
-  cy.visit(`${Cypress.env('opensearchDashboards')}/app/observability-dashboards#/application_analytics/`);
-  suppressResizeObserverIssue();
-  cy.wait(delay * 6);
+  cy.visit(`${Cypress.env('opensearchDashboards')}/app/observability-applications#/`);
+  cy.get('.euiTableRow').should('have.length.greaterThan', 0);//Replaces Wait
   cy.get(`[data-test-subj="${name}ApplicationLink"]`).click();
-  cy.wait(delay);
+  cy.get('.euiTableRow').should('have.length.lessThan', 1);//Replaces Wait
```

Comment:
what does less than 1 mean? You are looking for an empty table? Can we replace this to make it more clear?

Reply:
It is checking to see that that the clicking on the Application Link has fully loaded before searching for the name of the application. The table no longer exist on the new page so this ensures the loading completes and delays the next command "find applicationTitle" from occurring till it is on the page and able to be found.